src/ray/object_manager/plasma/create_request_queue.cc

```diff
@@ -138,12 +138,13 @@ Status CreateRequestQueue::ProcessRequests() {
                                 *request_it,
                                 /*spilling_required=*/nullptr);
         if (!status.ok()) {
+          (*request_it)->error = PlasmaError::OutOfDisk;
```

Comment:
Can you comment the reasoning behind why fallback allocation fails -> out of disk here?